<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PEP Assessor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
     QWidget {
     background-color: #8DB6D3;
     }
     QWidget[nonrelease=true], QWidget[nonrelease=true] * {
     background-color: #e67919;
     }
     QComboBox QAbstractItemView {
     selection-background-color: #8DB6D3;
     }
     QComboBox[nonrelease=true] QAbstractItemView[nonrelease=true] {
     selection-background-color: #e67919;
     }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>45</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="pepLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>200</height>
      </size>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>10</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="updateButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>10</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget {
    border: 0.1em solid white;
    border-radius: 0.25em;
    color: white;
    padding: 0.5em;
    outline: none;
    font-size: 13pt;
}
QWidget:focus {
	border: 0.2em solid white;
}
QWidget:hover {
	background-color: rgba(255,255,255,0.9);
	color: #8DB6D3;
}
QWidget:disabled {
	color: grey;
	border-color: grey;
}
</string>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="visible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="loginButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>10</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget {
    border: 0.1em solid white;
    border-radius: 0.25em;
    color: white;
    padding: 0.5em;
    outline: none;
    font-size: 13pt;
}
QWidget:focus {
    border: 0.2em solid white;
}
QWidget:hover {
    background-color: rgba(255,255,255,0.9);
    color: #8DB6D3;
}
QWidget:disabled {
    color: grey;
    border-color: grey;
}
</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="version">
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>LoginWidget</sender>
   <signal>version(QString)</signal>
   <receiver>version</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>version(QString)</signal>
 </slots>
</ui>
